src/GridField/GridFieldConfig_CustomRelationEditor.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace CodeCraft\Pathfinder\GridField;
+
+use SilverStripe\Forms\GridField\GridField_ActionMenu;
+use SilverStripe\Forms\GridField\GridFieldAddExistingAutocompleter;
+use SilverStripe\Forms\GridField\GridFieldButtonRow;
+use SilverStripe\Forms\GridField\GridFieldConfig;
+use SilverStripe\Forms\GridField\GridFieldDeleteAction;
+use SilverStripe\Forms\GridField\GridFieldDetailForm;
+use SilverStripe\Forms\GridField\GridFieldEditButton;
+use SilverStripe\Forms\GridField\GridFieldToolbarHeader;
+use SilverStripe\Versioned\Versioned;
+use SilverStripe\Versioned\VersionedGridFieldState\VersionedGridFieldState;
+use SilverStripe\Versioned\VersionedGridFieldStateExtension;
+
+/**
+ * A customised relation editor config for managing Pathfinder content
+ */
+class GridFieldConfig_CustomRelationEditor extends GridFieldConfig
```

Comment:
I was gonna say, interesting that the PHP linter didn't pick up the `_` format as it threw one for my `.._ItemRequest` class but since this is a module - makes sense that it didn't get picked up

Reply:
Yeah, not the most ideal naming convention. Hopefully, we see it enhanced in 5